// @vx/mock-data/build/index.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _genDateValue = require('./generators/genDateValue');

Object.defineProperty(exports, 'genDateValue', {
  enumerable: true,
  get: function get() {
    return _interopRequireDefault(_genDateValue).default;
  }
});

var _genRandomNormalPoints = require('./generators/genRandomNormalPoints');

Object.defineProperty(exports, 'genRandomNormalPoints', {
  enumerable: true,
  get: function get() {
    return _interopRequireDefault(_genRandomNormalPoints).default;
  }
});

var _genBin = require('./generators/genBin');

Object.defineProperty(exports, 'genBin', {
  enumerable: true,
  get: function get() {
    return _interopRequireDefault(_genBin).default;
  }
});

var _genBins = require('./generators/genBins');

Object.defineProperty(exports, 'genBins', {
  enumerable: true,
  get: function get() {
    return _interopRequireDefault(_genBins).default;
  }
});

var _genStats = require('./generators/genStats');

Object.defineProperty(exports, 'genStats', {
  enumerable: true,
  get: function get() {
    return _interopRequireDefault(_genStats).default;
  }
});

var _appleStock = require('./mocks/appleStock');

Object.defineProperty(exports, 'appleStock', {
  enumerable: true,
  get: function get() {
    return _interopRequireDefault(_appleStock).default;
  }
});

var _letterFrequency = require('./mocks/letterFrequency');

Object.defineProperty(exports, 'letterFrequency', {
  enumerable: true,
  get: function get() {
    return _interopRequireDefault(_letterFrequency).default;
  }
});

var _browserUsage = require('./mocks/browserUsage');

Object.defineProperty(exports, 'browserUsage', {
  enumerable: true,
  get: function get() {
    return _interopRequireDefault(_browserUsage).default;
  }
});

var _groupDateValue = require('./mocks/groupDateValue');

Object.defineProperty(exports, 'groupDateValue', {
  enumerable: true,
  get: function get() {
    return _interopRequireDefault(_groupDateValue).default;
  }
});

var _cityTemperature = require('./mocks/cityTemperature');

Object.defineProperty(exports, 'cityTemperature', {
  enumerable: true,
  get: function get() {
    return _interopRequireDefault(_cityTemperature).default;
  }
});

var _lesMiserables = require('./mocks/lesMiserables');

Object.defineProperty(exports, 'lesMiserables', {
  enumerable: true,
  get: function get() {
    return _interopRequireDefault(_lesMiserables).default;
  }
});

var _exoplanets = require('./mocks/exoplanets');

Object.defineProperty(exports, 'exoplanets', {
  enumerable: true,
  get: function get() {
    return _interopRequireDefault(_exoplanets).default;
  }
});

var _planets = require('./mocks/planets');

Object.defineProperty(exports, 'planets', {
  enumerable: true,
  get: function get() {
    return _interopRequireDefault(_planets).default;
  }
});

var _shakespeare = require('./mocks/shakespeare');

Object.defineProperty(exports, 'shakespeare', {
  enumerable: true,
  get: function get() {
    return _interopRequireDefault(_shakespeare).default;
  }
});

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
